lib/Screens/AboutUs.dart

```diff
@@ -1,10 +1,355 @@
+
+  Future<void> _fetchContributors() async {
+    const contributorsAPIUrl =
+        'https://api.github.com/repos/Saransh-cpp/BookRentApp-Chapter3/contributors';
+
+    final response = await http.get(Uri.parse(contributorsAPIUrl));
+
+    if (response.statusCode == 200) {
+      setState(() {
+        jsonContributors = jsonDecode(response.body) as List;
+      });
+    } else {
+      Fluttertoast.showToast(
+        msg: "Failed to load Contributors.",
+        toastLength: Toast.LENGTH_SHORT,
+        gravity: ToastGravity.BOTTOM,
+      );
+    }
+  }
+
+  bool _isLoading = true;
+
+  @override
+  void initState() {
+    super.initState();
+    _fetchContributors();
+    Future.delayed(const Duration(seconds: 2), () {
+      setState(() {
+        _isLoading = false;
+      });
+    });
+  }
 
-class AboutUs extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      body: _isLoading
+          ? const SpinKitRotatingCircle(
+              color: Colors.blue,
+            )
+          : SafeArea(
+              child: SingleChildScrollView(
+              child: Container(
+                margin: const EdgeInsets.only(top: 30),
+                child: Column(
+                  children: <Widget>[
+                    Container(
+                      alignment: Alignment.topCenter,
+                      margin: EdgeInsets.only(
+                        top: 10,
+                      ),
+                      child: Row(
+                        mainAxisSize: MainAxisSize.min,
+                        children: <Widget>[
+                          const Text(
+                            "ABOUT",
+                            style: TextStyle(
+                                fontSize: 25, fontWeight: FontWeight.w700),
+                          ),
+                          Container(
+                            margin: const EdgeInsets.only(left: 8),
+                            child: const Text(
+                              "Chapter 3",
+                              style: TextStyle(
+                                fontSize: 25,
+                                fontWeight: FontWeight.w700,
+                                // color: Colors.orange[900],
+                              ),
+                            ),
+                          )
+                        ],
+                      ),
+                    ),
+                    Container(
+                      padding:
+                          EdgeInsets.symmetric(vertical: 10, horizontal: 15),
+                      child: Card(
+                        elevation: 1,
+                        shadowColor: Colors.grey,
+                        child: Padding(
+                          padding: EdgeInsets.all(20),
+                          child: Text(_about_this_app),
```

Comment:
```suggestion
                          child: Text(_aboutThisApp),
```

Can you also add center align here?